spec/features/directly_comment_on_post_spec.rb

```diff
@@ -0,0 +1,40 @@
+require "rails_helper"
+
+feature "Comment on a Post" do
+  let(:user) { create(:oliver) }
+  let(:post) { create(:url_post,
+    title: "Y Combinator Is Launching A “Grad School” For Booming Startups") }
+
+  it "allows authenticated user to comment on a post and see new comment on the parent post show view" do
+    submitter = user
+    sign_in submitter
+
+    visit post_path((post).id)
+    expect(page).to have_http_status(200)
```

Comment:
can probably remove this line as a 200 can be implied by the content expectations below.